quality-assurance/acceptance-criteria.md

```diff
@@ -4,21 +4,33 @@ This page provides tips on writing effective acceptance criteria (AC).
 
 ## What is Acceptance Criteria
 
-> Conditions that a software product must satisfy to be accepted by a user, customer or other stakeholder.
+> Conditions that a software product must satisfy to be accepted by a user, customer or other stakeholder. 
+> They are confirmation that the requirements captured in the description have been achieved, and used as part 
+> of the test criteria also
 
 ## Why write good Acceptance Criteria
 
-To ensure all members of a team have a shared understanding of the requirements of the stories they are developing.
+To ensure all members of a team have a shared understanding of the requirements of the stories they are developing and what needs to be tested 
+in order to confirm that the story is DONE
 
 ## How do I know I am doing it right
 
 1. The whole team is involved in the creation and refinement of AC
-1. AC is recorded and agreed before a user story is played
-1. Every user story has at least 1 AC - too many ACs? Split the story
-1. All criteria are independently testable with a clear pass / fail outcome
-1. Non-functional criteria is included where applicable
+2. they should be clear and unambigious to ensure clarity
+3. AC is recorded and agreed before a user story is played
+4. Every user story has at least 1 AC - too many ACs? Split the story
+5. All criteria are independently testable with a clear pass / fail outcome
+6. Non-functional criteria is included where applicable
 
 ## How to generate good Acceptance Criteria
 
 1. Use Example Mapping sessions - [Example Mapping as part of BDD](bdd.md)
 1. Record it using Gherkin scenario-orientated syntax - “Given...When...Then...”
+2. Please remember these are behaviour driven and singlular so they should not be trying to depict multible out interations
```

Comment:
Need to begin all items with a 1.
Singlular.  Spelling, and also what does this mean?
multible out interations (spelling, also what does this mean?)